code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -642,20 +662,20 @@ nobiliumsuppression = INFINITY
 	var/list/cached_gases = air.gases
 	var/temperature = air.temperature
 	var/old_heat_capacity = air.heat_capacity()
-	var/heat_efficency = min(temperature/(FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 10), cached_gases[/datum/gas/plasma][MOLES], cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/bz][MOLES])
+	var/heat_efficency = min(temperature / (FIRE_MINIMUM_TEMPERATURE_TO_EXIST * 10), cached_gases[/datum/gas/plasma][MOLES], cached_gases[/datum/gas/carbon_dioxide][MOLES], cached_gases[/datum/gas/bz][MOLES])
 	var/energy_used = heat_efficency * 100
-	ASSERT_GAS(/datum/gas/freon,air)
-	if ((cached_gases[/datum/gas/plasma][MOLES] - heat_efficency < 0 ) || (cached_gases[/datum/gas/carbon_dioxide][MOLES] - heat_efficency < 0) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
+	ASSERT_GAS(/datum/gas/freon, air)
+	if ((cached_gases[/datum/gas/plasma][MOLES] - heat_efficency * 1.5 < 0 ) || (cached_gases[/datum/gas/carbon_dioxide][MOLES] - heat_efficency * 0.75 < 0) || (cached_gases[/datum/gas/bz][MOLES] - heat_efficency * 0.25 < 0)) //Shouldn't produce gas from nothing.
 		return NO_REACTION
-	cached_gases[/datum/gas/plasma][MOLES] -= heat_efficency * 5
-	cached_gases[/datum/gas/carbon_dioxide][MOLES] -= heat_efficency
+	cached_gases[/datum/gas/plasma][MOLES] -= heat_efficency * 1.5
+	cached_gases[/datum/gas/carbon_dioxide][MOLES] -= heat_efficency * 0.75
 	cached_gases[/datum/gas/bz][MOLES] -= heat_efficency * 0.25
-	cached_gases[/datum/gas/freon][MOLES] += heat_efficency * 2
+	cached_gases[/datum/gas/freon][MOLES] += heat_efficency * 2.5
```

Comment:
more changes?

Reply:
same reason as before, the gases are too much used without a good output of products, this way i also balanced the reaction a bit (1.5+0.75+0.25 = 2.5)